be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/ElectionValidator.scala

```diff
@@ -348,6 +387,30 @@ sealed class ElectionValidator(dbActorRef: => AskableActorRef) extends MessageDa
       case _      => Right(rpcMessage)
     }
 
+  /** Does the opposite of the above function, i.e : checks if the election has ended
```

Comment:
I think you can shorten that to:
```suggestion
  /** checks if the election has ended
```